styles.css

```diff
@@ -0,0 +1,146 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap');
+
+.first-page {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+  background: #fff;
+}
```

Comment:
- [OPTIONAL] The first-page section is missing some background images from the Figma template. It would be great to see the images there.